client/components/mma/identity/form/FormField.tsx

```diff
@@ -90,7 +90,7 @@ const getInputFieldOfType = (type: string) => {
 			label={props.label}
 			formikProps={props.formikProps}
 		>
-			<Field type={type} />
+			<Field type={type} data-qm-masking="blocklist" />
```

Comment:
what does this component display?

Reply:
It's the forms used on the Setting and Profile pages